.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea
```

Comment:
Please don't do this. User-specific ignores belong into user-specific global git configuration, not into projects.

Reply:
understand. Thank you.